pkg/services/ngalert/api/api_testing_test.go

```diff
@@ -41,15 +143,14 @@ func TestRouteTestGrafanaRuleConfig(t *testing.T) {
 				{Action: datasources.ActionQuery, Scope: datasources.ScopeProvider.GetResourceScopeUID(data1.DatasourceUID)},
 			})
 
-			srv := createTestingApiSrv(nil, ac, nil)
+			srv := createTestingApiSrv(t, nil, ac, eval_mocks.NewEvaluatorFactory(&eval_mocks.ConditionEvaluatorMock{}))
 
-			response := srv.RouteTestGrafanaRuleConfig(rc, definitions.TestRulePayload{
-				Expr: "",
-				GrafanaManagedCondition: &definitions.EvalAlertConditionCommand{
-					Condition: data1.RefID,
-					Data:      ApiAlertQueriesFromAlertQueries([]models.AlertQuery{data1, data2}),
-					Now:       time.Time{},
-				},
+			rule := validRule()
+			rule.GrafanaManagedAlert.Data = ApiAlertQueriesFromAlertQueries([]models.AlertQuery{data1, data2})
+			response := srv.RouteTestGrafanaRuleConfig(rc, definitions.PostableExtendedRuleNodeExtended{
```

Comment:
The signature of `RouteTestGrafanaRuleConfig()` changed, so I need a `PostableExtendedRuleNodeExtended` now, not a `TestRulePayload`